Makefile

```diff
@@ -33,7 +33,7 @@ fetch-dependencies:
 	go get github.com/captncraig/caddy-realip
 	go get gopkg.in/natefinch/lumberjack.v2
 	go get github.com/miekg/dns
-	go get -d -u
```

Comment:
I don't know why we had that as a final `go get` step before, should it be removed now ?

Reply:
To keep all the code updated. Go get doesn't update dependencies, when they are already available.

Reply:
So that should probably stay then :)